src/components/PatchStatusBadge.tsx

```diff
@@ -0,0 +1,27 @@
+import React from "react";
+import { PatchStatus } from "types/patch";
```

Comment:
Is there a generated type we can use thats already available inside of `generated/types` ?

Reply:
tldr: no. this is a global evergreen type representing the different patch statuses as an enum. it's not in generated types not do i think it should it be.

Reply:
It would make sense to use a generated enum it existed but it doesn't exist since our Patch gql type status field is a string.

Reply:
Thanks didn't know it wasnt.